src/elexmodel/utils/math_utils.py

```diff
@@ -2,6 +2,14 @@
 from scipy.stats import bootstrap
 
 
+class ModelClientException(Exception):
```

Comment:
we already define a `ModelClientException` in `client.py`. I'm not 100% sure what the implications of doing this twice is. It might be good to move the exceptions to an `exceptions.py` file and then use them in all the places we need them. Curious what you think @emilyliu7321?

Reply:
I agree with you!